### DISCUSSION TITLE
Sega CD Backup RAM cart not dumping correctly

### DISCUSSION THREAD

**Discussion by jnftech** [sanni/cartreader#331] [Need help]
I have an original CD Backup RAM cart that does not dump correctly. I am using V3 hardware with release 7.3.
The unit will dump anything else I have thrown at it - SNES, Genesis, N64, GB/GBA, NES (with adapter), etc.
The dump process completes and writes to a file, but the file comes out as repeating 0x04 through the entire 128k file (See attachment). This cart does work and has data if used with a SegaCD.
Does anyone else have this issue?

**Comment by whitezombie2000** [comment]
I just got my Sanni Cart Reader today and tested mine using v8.3. It worked just fine and I had no issues. I was able to rename the the dumped Cart.brm file to a .crm file and all the saves were available in Kega Fusion.

**Reply by jnftech** [reply]
Thanks for the reply. I just tried updating to version 9.4 and the results are the same. The file is effectively blank (like the one attached above) but the cart shows contents in a Sega CD.
What hardware version of the cart reader are you using?

**Comment by whitezombie2000** [comment]
I am using Cart Reader v3.
